<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KProductManagerWindow</class>
 <widget class="QWidget" name="KProductManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1">
   <item>
    <widget class="QTableView" name="m_table_product"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="5,2">
     <item>
      <layout class="QFormLayout" name="product_info">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>类别</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="m_product_category">
         <item>
          <property name="text">
           <string>其他</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>电脑办公</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>智能手机</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>食物饮料</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>服装鞋靴</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>名称</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="m_product_name"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>数量</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="m_product_number">
         <property name="maximum">
          <number>10000000</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>进价</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>售价</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>样图</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="m_product_purchase_price">
         <property name="prefix">
          <string>¥</string>
         </property>
         <property name="maximum">
          <double>9999999999.989999771118164</double>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="m_product_selling_price">
         <property name="prefix">
          <string>¥</string>
         </property>
         <property name="maximum">
          <double>9999999999.989999771118164</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="m_poduct_picture">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>120</height>
          </size>
         </property>
         <property name="text">
          <string>picture</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="m_insert_product_btn">
           <property name="text">
            <string>添加商品</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_delete_product_btn">
           <property name="text">
            <string>删除商品</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_setpic_btn">
           <property name="text">
            <string>设置图片</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="m_save_product_btn">
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_clear_product_btn">
           <property name="text">
            <string>清仓</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
